canvas_modules/common-canvas/src/common-properties/components/truncated-content-tooltip/truncated-content-tooltip.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.properties-readonly .tooltip-trigger,
```

Comment:
Why do we need each control classname?  Should we have a new classname for `className="properties-tooltips"` above in the new component?  Then use that classname here.  Then we don't need to keep adding controls as the new component gets used.

Reply:
`className="properties-tooltips"` used in 2 places in `TruncatedContentTooltip` component. 
Updated code - 
```
	const tooltip = (
			<div className="properties-tooltips123">
				{String(this.props.tooltipText)}
			</div>
		);
		return (<Tooltip
			id={`${uuid4()}-properties`}
			tip={tooltip}
			direction="bottom"
			className="properties-tooltips456"
			disable={has(this.props, "disabled") ? this.props.disabled : true}
			showToolTipIfTruncated
		>
			{this.props.content}
		</Tooltip>);
```
![Screen Shot 2021-12-02 at 5 30 49 PM](https://user-images.githubusercontent.com/25124000/144529503-9e5ef7b9-60a9-4513-814d-8faf96e46de6.png)


`.tooltip-trigger` class is under `.properties-fp-field` - 
![Screen Shot 2021-12-02 at 5 31 49 PM](https://user-images.githubusercontent.com/25124000/144529708-d5bad2ce-5640-4e55-a2b0-adeac30c555a.png)

Reply:
I'm a little confused by this statement.  I don't know why we need `.properties-readonly .tooltip-trigger ...` since we already have a truncated-content-tooltip component that could be the parent class for all these.  We should be able to have something like `.properties-truncated-tooltip .tooltip-tipper` as the only classes.